cx_Freeze/hooks.py

```diff
@@ -677,9 +677,10 @@ def load_win32com(finder, module):
 
 
 def load_win32file(finder, module):
-    """the win32api module implicitly loads the pywintypes module; make sure
-       this happens."""
+    """the win32api module implicitly loads the pywintypes and win32timezone
+       module; make sure this happens."""
     finder.IncludeModule("pywintypes")
+    finder.IncludeModule("win32timezone")
```

Comment:
Here is an example for you :-)
I modified minor things of the wxPython example supplied with the cx_freeze package. You can compare these for seeing the difference.

If you run the setup.py for this example you will first see the error described in issue #254. If you patch cx_freeze with the pull request #260, the script runs perfect.

After freezing, start the executable and press the button "Push me". You will get the error described in the issue #212 => `ModuleNotFoundError: No module named 'win32timezone'`

My environment: Win 7/10 64Bit, Python 3.6.1, pywin32 and wxPython 4.0.0a1.
Attached example: [cx_freeze-bug_test.zip](https://github.com/anthony-tuininga/cx_Freeze/files/1005069/cx_freeze-bug_test.zip)